README.md

```diff
@@ -56,6 +57,12 @@ grunt.initConfig({
 
 Execute with `grunt jsinspect:examples`.
 
+Please note that in case the configuration file exists and its name matches the default value
+`.jsinspectrc`, it will be used to override any other configuration option that might be present
```

Comment:
Uhhhh, thanks... This fills a spelling education gap on my side. Thanks. :dancer: